kubernetes/windows/Dockerfile

```diff
@@ -3,7 +3,7 @@ MAINTAINER OMSContainers@microsoft.com
 LABEL vendor=Microsoft\ Corp \
     com.microsoft.product="Azure Monitor for containers"
 
-ARG IMAGE_TAG=win-ciprod04222021
+ARG IMAGE_TAG=win-ciprod05122021

```

Comment:
since we are not making updates to windows , as discussed, we can skip the windows image related updates.